pkg/controller/garbage_collector_test.go

```diff
@@ -101,7 +101,7 @@ func TestGarbageCollector_StartGC(t *testing.T) {
 		DeleteCollectionCb: func(options *v1.DeleteOptions, listOptions v1.ListOptions) error {
 			assert.NotNil(t, options)
 			assert.NotNil(t, listOptions)
-			assert.Equal(t, "hour-of-day in (0,1,10,11,12,13,14,15,16,17,18,19,2,20,21,3,4,5,6,7,8,9),termination-status=terminated", listOptions.LabelSelector)
+			assert.Equal(t, "hour-of-day notin (10.22,10.23),termination-status=terminated", listOptions.LabelSelector)
```

Comment:
```suggestion
			assert.Equal(t, "hour-of-day not in (10.22,10.23),termination-status=terminated", listOptions.LabelSelector)
```

Reply:
Hi @pingsutw, thanks for your review. 
according to [k8s doc](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/):
Set-based label requirements allow filtering keys according to a set of values. Three kinds of operators are supported: in,notin and exists